templates/tools/dockerfile/valgrind.include

```diff
@@ -0,0 +1,7 @@
+#=================
```

Comment:
nit: if the file contains PHP test dependencies, then it should be called sth like `php_valgrind.include` instead.